### DISCUSSION TITLE
API Reference on readthedocs.io is empty

### DISCUSSION THREAD

**Discussion by loelkes** [miguelgrinberg/Flask-SocketIO#1995] [General]
The API reference at [https://flask-socketio.readthedocs.io/en/latest/api.html](https://flask-socketio.readthedocs.io/en/latest/api.html) gives me

<img width="982" alt="Screenshot 2023-06-13 at 11 22 55" src="https://github.com/miguelgrinberg/Flask-SocketIO/assets/231909/9e12f118-b399-4fdb-83dd-31c05e2ed988">

If I build the docs manually in the repo the page shows the expected content. I didn't file an issue since it doesn't seem to be an "issue with the repo"...

**Comment by miguelgrinberg** [comment]
Thanks for the alert. I have rebuilt the docs and the reference API is now included.
